constants.h

```diff
@@ -1,14 +1,17 @@
 #ifndef CONSTANTS_H
 #define CONSTANTS_H
 
+#include <cmath>
+
 namespace constants
 {
     const double kAccuracy = 0.00000001;
     const double kSoundSpeed = 343.216; // This is measured in 20'C at 1000 hPa
-    const double kPi = 3.1415926535897932384626433832795;
+    const double kPi = std::acos(-1);
     const double kSimulationRadius = 4; // [m]
     const int kPopulation = 37;
     const double kHitAccuracy = 0.0001;
+    const size_t kCollectors = 37;
```

Comment:
site_t -> int

While I understand where this is coming from, size_t may be easy to misuse, and sometimes can cause errors. 

Guess (without running it) what the output of this program will be:

#include <iostream>

int main() {
  size_t num_collectors = 10;
  if (num_collectors - 20 > 0) {
    std::cout << "Still have some collectors! That many" << num_collectors - 20 << std::endl;
  } else {
    std::cout << "no more collectors!" << std::endl;
  }
}

Reply:
Im guessing that output will be: " Still have some collectors ! That many WEIRD_NUMEBR" ... size_t cant be less then 0

Reply:
Bingo:

```
Still have some collectors! That many18446744073709551606
```